### ISSUE TITLE
Améliorations de la navigation dans les marées et par dates

### ISSUE THREAD

**Issue by VincentAntoine** [MTES-MCT/monitorfish#3563]
Retour MEMN

> Par exemple: je veux connaitre les détails des infractions du contrôle du navire XXX du 17/10/2023 qui ressort en PEL01/02 dans mon extraction Métabase.
>
> Une fois sur la fiche MonitorFish de mon inspection avec infraction, le niveau de détail peut différer. Lorsqu'un commentaire est glissé sous chaque NATINF pour expliquer les choses, c'est le TOP mais ce n'est pas le cas tout le temps. A contrario, parfois il n'y a pas d'explications. Comme l'exige l'EFCA pour les PEL, je dois m'assurer que je suis bien sur OTM/PTM avec une atteinte sur HER/MAC/HOM/PIL/WHB. Je dois donc vérifier la marée et les messages du JPE. Mais il n'y a **pas de lien entre la fiche de contrôle et la marée correspondante** -> est-ce que ce lien direct par un clic peut être fait?
>
> Je cherche donc la piste VMS par les dates du contrôle -> les messages ne s'affichent pas sur la piste quand je clique sur l'option et la marée correspondant à la piste VMS ne s'affiche pas non plus.
Pour trouver la marée, je joue avec les numéros de marée pour essayer de tomber sur le bon mois et remonter sur le jour de mon infraction.
> Ce parcours de recherche est fastidieux et chronophage. 

- [ ] Accéder à la marée correspondante à partir d'un contrôle
- [x] Chercher les messages JPE correspondants quand on affiche les positions du navire par plage de dates

**Comment by louptheron** [comment]
@AdelineCelier J'ai eu un retour de ce type il y a quelques temps, on fait ces deux fonctionnalités ? ça ne devra pas être trop long je pense.

**Comment by AdelineCelier** [comment]
Yes carrément ! Je peux dessiner quelque chose en début de semaine pro si ça te va.

**Comment by AdelineCelier** [comment]
@louptheron j'ai rajouté un bouton "voir la marée du contrôle" à côté du bouton "voir le CR du contrôle" dans l'onglet des contrôles du navire, cf. [cet écran](https://xd.adobe.com/view/415a5b72-b2a2-4dcb-8d51-36b7af809d90-1409/screen/6d11df38-f893-455e-b618-83ca80c1412c/).

**Comment by louptheron** [comment]
Ok en inté pour le 2ème point.
